Proposal: recommend defining a @VERSION_SUFFIX@ token in addition to a @TOOL_VERSION@ token (#59)
galaxy-iuc/standards: docs/best_practices/tool_xml.rst

- define a ``@TOOL_VERSION@``
  `macro token <https://planemo.readthedocs.io/en/latest/writing_advanced.html#macro-tokens>`__,
  which you should also re-use in the corresponding ``<requirement>`` element;

> - optionally, define a ``@VERSION_SUFFIX@`` macro token, which may be placed either
>   in the tool wrapper or in a shared macro file. This should be set to:
> 
>   - 0 for the first wrapper release of each version of the underlying tool.
>   - an integer number to be increased by 1 whenever you update the wrapper
>     without changing the underlying ``@TOOL_VERSION@``.

Reviewer: ```suggestion
- optionally, define a ``@VERSION_SUFFIX@`` macro token, which may be placed
  either in the tool wrapper or in a shared macro file. This should be set to
  an integer number:

  - starting at 0 for the first wrapper release of each version of the
    underlying tool;
  - to be increased by 1 whenever you update the wrapper without changing
    the underlying ``@TOOL_VERSION@``.
```

- define a ``@TOOL_VERSION@``
  `macro token <https://planemo.readthedocs.io/en/latest/writing_advanced.html#macro-tokens>`__,
  which you should also re-use in the corresponding ``<requirement>`` element;
- optionally, define a ``@VERSION_SUFFIX@`` macro token, which may be placed
  either in the tool wrapper or in a shared macro file. This should be set to
  an integer number:

  - starting at 0 for the first wrapper release of each version of the
    underlying tool;
  - to be increased by 1 whenever you update the wrapper without changing
    the underlying ``@TOOL_VERSION@``.

> - set the tool ``version`` attribute to ``@TOOL_VERSION@+galaxy@VERSION_SUFFIX@``. If it

Reviewer: ```suggestion
- set the tool ``version`` attribute to ``@TOOL_VERSION@+galaxy@VERSION_SUFFIX@``. If it
```

>   is preferred not to use a ``@VERSION_SUFFIX@`` token, the version suffix can be specified
>   by simply using an integer ``N``, in which case the tool ``version`` attribute should be set
>   to ``@TOOL_VERSION@+galaxyN``.

Reviewer: ```suggestion
  is preferred not to use a ``@VERSION_SUFFIX@`` token (e.g. to allow bumping
  the version only for a specific tool in a suite), the tool ``version``
  attribute should be simply set to ``@TOOL_VERSION@+galaxyN``, where N is an
  integer following the same rules of ``@VERSION_SUFFIX@``.
```
